docs/configure_open_edx.md

```diff
@@ -23,6 +23,20 @@ There are two options for this:
 - In devstack, run `make lms-shell` and within that shell `pip install social-auth-mitxpro-$VERSION.tar.gz`
   - To update to a new development version without having to actually bump the package version, simply `pip uninstall social-auth-mitxpro`, then install again
 
+#### Install `mitxpro-openedx-extensions` in LMS
```

Comment:
Above this in the `Setup Open edX Devstack` section, mention that this should be set:
```
export OPENEDX_RELEASE=ironwood.master
```